sunflower-land/sunflower-land
# [FIX] Increase mailbox clickable area

# Description

- Increase clickable area for mailbox (mailbox hitbox was too small to click on mobile).  Lock bumpkin, airdrop and mailbox hitboxes to the grids:
**Clickable areas**
![image](https://user-images.githubusercontent.com/107602352/209082225-b03e9b1a-2ed2-49da-bbda-a1a7636ba8ce.png)

- standardize pixel sizes
- use newer panel components for bumpkin, airdrop and mailbox modals
- code refactoring

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- hover around bumpkin, airdrop and mailbox to check interactable areas
- click and feed bumpkin
- click and claim/close airdrop
- click and close mailbox

# Checklist:
